pizza-time/src/styles/navStyles.js

```diff
@@ -1,11 +1,33 @@
-import styled from 'styled-components';
+import styled, { keyframes } from 'styled-components';
```

Comment:
Had no idea keyframes was something in styled components. I'm guessing it allows you to abstract keyframe animations and drop them where necessary?